containers/webhooks.py

```diff
@@ -22,8 +21,20 @@ def release(url, tag):
             repo = git.Repo.clone_from(url, wd)
             repo.git.checkout(tag)
             sys.path.insert(0, wd)
-            distutils.core.run_setup(os.path.join(wd, 'setup.py'),
-                                     ['sdist', 'bdist_wheel', '--universal'])
+
+            # Run setup.py sdist bdist_wheel --universal
+            setup_path = os.path.join(wd, 'setup.py')
+            argv_backup = sys.argv.copy()
+            global_vars = {'__file__': setup_path}
+            try:
+                sys.argv[0] = setup_path
+                sys.argv[1:] = ['sdist', 'bdist_wheel', '--universal']
+                with open(setup_path, 'rb') as f:
+                    exec(f.read(), global_vars)
+            except SystemExit:
+                pass
+            finally:
+                sys.argv = argv_backup
```

Comment:
Couldn't we perhaps use `setuptools.sandbox.run_setup()`?

https://github.com/pypa/setuptools/blob/38974172d0ff7ead2a2b212c92f303438bcdf1a2/setuptools/sandbox.py#L241-L242

Reply:
Thanks, I didn't find an equivalent but this seems to be exactly what we need